news/2024/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: post
+title: "The Followpoint: Predominador, the Grandmaster of SS-ranks"
+date: 2024-10-21 19:00:00 +0000
+series: The Followpoint
+---
+
+Have you ever wondered what is going on inside a score farmer's head? Well, today is your chance to find out in our latest interview!
+
+![](/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/banner.jpg)
+
+osu!catch players, rejoice, as we are releasing an interview with a figure many of you may know, [Predominador](https://osu.ppy.sh/users/4568537)! There are quite a few reasons you may have heard of them, from their [Beatmap Spotlights](/wiki/Beatmap_Spotlights) participation to reaching insane milestones, one of such being 100,000 SS ranks, which they achieved recently! And in this interview we can learn more about Predominador and their thoughts on some aspects of the game. 👀
+
+Now, let's get to reading!
+
+---
+
+<style>
+    .news-chat-quote__avatar {
+        float: left;
+        width: 40px;
+        height: 40px;
+        border-radius: 50%;
+        margin-left: -50px;
+    }
+
+    .news-chat-quote__text-container {
+        margin-left: 50px;
+    }
+
+    .news-chat-quote__username {
+        font-weight: 600;
+        margin-bottom: 2px;
+    }
+
+    .news-chat-quote__colour-lvd {
+        color: #FFD1DC;
+    }
+
+    .news-chat-quote__colour-no-group {
+        color: #FFFFFF;
+    }
+</style>
+
+<div class="news-chat-quote__text-container">
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Hi Predominador, thanks for joining us today! Just like in our other interviews, could you please introduce yourself for those who might not know you?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4568537" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-Predominador.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4568537">Predominador</a></p>
+
+Hello! I am Predominador, from Brazil. I am a SS farmer in osu!catch, who recently reached 100k SS ranks! Besides SS, I also like to try to get some first places in songs I like (or any other good placement), and from time to time to try to get new medals, too! Since a decade ago, I have been adventuring myself in all game modes, and like to play all of them, but currently I am mostly playing only osu!catch! My favorite mods are HD and FL.
```

Comment:
```suggestion
Hello! I am Predominador, from Brazil. I am a SS farmer in osu!catch, who recently reached 100k SS ranks! Besides SS, I also like to try to get some first places in songs I like (or any other good placement), and from time to time to try to get new medals, too! Since a decade ago, I have been adventuring myself in all gamemodes, and like to play all of them, but currently I am mostly playing only osu!catch! My favorite mods are HD and FL.
```

Reply:
it's always styled as `game modes` in wiki/news.